pkg/termination/termination_suite_test.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package termination
```

Comment:
Does IBM have a concept of spot/preemptible instances? Any links?

Reply:
We do have something called `transient vs`, but I personally have not tried provisioning one. 

Also, from what I know, it was offered in the previous generation of VPC (Classic Infra). I don't think it's there in VPC Gen2.

Reply:
If that's the case, I'd suggest removing the termination handler stuff for now, this is all to do with spot instances. We can add support for this later